WearVerifyRemoteApp/Application/build.gradle

```diff
@@ -1,70 +1,52 @@
+/*
+ * Copyright (C) 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id "com.android.application"
+    id "kotlin-android"
+}
 
-buildscript {
-    repositories {
-        google()
-        jcenter()
-    }
+android {
+    compileSdkVersion 30
 
-    dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.0'
+    defaultConfig {
+        versionCode 1
+        versionName "1.0"
+        minSdkVersion 26
+        targetSdkVersion 30
     }
-}
 
-apply plugin: 'com.android.application'
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
 
-repositories {
-    google()
-    jcenter()
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
 }
 
 dependencies {
-
-
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.legacy:legacy-support-v13:1.0.0'
     implementation 'androidx.cardview:cardview:1.0.0'
     implementation 'androidx.appcompat:appcompat:1.2.0'
-
     implementation 'com.google.android.gms:play-services-wearable:17.0.0'
-
     implementation 'com.google.android.support:wearable:2.8.1'
     compileOnly 'com.google.android.wearable:wearable:2.8.1'
```

Comment:
We are moving to androidx, right? I wasn't sure if this was the first change and then going to androidx?

If you aren't using any of the wearable support library, I would remove these.

Reply:
Correct! This was the first change just to add the Kotlin plugin, I'll be removing the support library references in a future PR.